content/recipes/infrastructure/gupri.md

```diff
@@ -0,0 +1,157 @@
+(fcb-infra-gupri)=
+# Creating resolvable identifiers
+
+````{panels_fairplus}
+:identifier_text: 
+:identifier_link: 
+:difficulty_level: 2
+:recipe_type: hands_on
+:reading_time_minutes: 30
+:intended_audience: data_manager, data_scientist  
+:has_executable_code: nope
+:recipe_name: GUPRIs
+```` 
+
+
+## Background knowledge required:
+
+* HTTP
+* URL
+* DNS
+
+
+## Abstract
+
+Missing
+
```

Comment:
Adding the aim of this recipe:
The importance of Globally Unique, Persistent, Resolvable Identifier and introducing a few public services for minting GUPRIs.